pfcpiface/messages.go

```diff
@@ -239,12 +246,15 @@ func (pc *PFCPConn) handleSessionEstablishmentRequest(upf *upf, msg message.Mess
 		sendError(errors.New("Unable to allocate new PFCP session"))
 	}
 	session := pc.mgr.sessions[localSEID]
-
+	var fseidIP uint32
+	_ = binary.Read(bytes.NewBuffer(net.ParseIP(sourceIP).To4()),
+		binary.LittleEndian, &fseidIP)
```

Comment:
dont understand the need for this in every pdr and far. it makes sense at a session level, we can store it there as we already store local and remote SEID.

Reply:
the key used for PDR and FAR is made up of fseid and fseidIP. that's why adding to the pdr and far. cause we pass the far and pdr structures to the fast path below.

Reply:
ok. should we extract it this way?
https://pkg.go.dev/github.com/wmnsk/go-pfcp@v0.0.8/ie#FSEIDFields

we already do 
```
	/* Read fseid from the IE */
	fseid, err := sereq.CPFSEID.FSEID()
	if err != nil {
		log.Println("Failed to parse FSEID from session establishment request")
		return nil
	}
	remoteSEID := fseid.SEID
```
so we can do this?
```
p.fseidIP = ip2int(fseid.IPv4Address)
```

Reply:
done